_data/engine-cli/docker_ps.yaml

```diff
@@ -521,7 +521,7 @@ examples: |-
     To list all running containers in JSON format, use the `json` directive:
 
     ```console
-    $ docker ps --format json
+    $ docker ps --format '{{json .}}'
```

Comment:
The example is correct, but requires docker 23.0 or up. If you're using docker desktop, you may still be on the previous (docker 20.10) version, which doesn't support this option